app/models/configuration_script_source.rb

```diff
@@ -4,4 +4,8 @@ class ConfigurationScriptSource < ApplicationRecord
   belongs_to  :manager, :class_name => "ExtManagementSystem"
 
   virtual_total :total_payloads, :configuration_script_payloads
+
+  def self.display_name(number = 1)
+    n_('Repository', 'Repositories', number)
```

Comment:
Note that this term may not apply to other variants in the future, which is why we made the model name generic.  It works for Ansible, but that's not the only thing that can use this table.